tdrs-backend/clamav-router/nginx.conf

```diff
@@ -0,0 +1,20 @@
+events { worker_connections 1024;
+}
+
+
+http{ 
+    server {
+        listen {{port}};
```

Comment:
We really do not need the first server proxy, however the nginx config need to have the default {{port}} config, otherwise it throws error, and unfortunately there is no way to change the {{port}} environment variable in cloud.gov.